site/content/blog/2019-06-05-ldap-nested-groups-modelling-and-representation-in-code.md

```diff
@@ -0,0 +1,119 @@
+---
+title: "LDAP Nested Groups: Modelling and Representation in Code"
+author: Martin Kraft
+date: 2019-06-05T12:00:00-04:00
+github: mkraft
+community: martin.kraft
+---
+
+## Defining Nested Groups
+
+The two main types of groups in LDAP are `groupOfNames` and `groupOfUniqueNames`. At minimum they have a `cn` (common name) attribute and can have membership attributes `member` or `uniqueMember`, respectively. 
```

Comment:
I'm wondering if we'd add any background or context on what this is related to -- e.g. a link to LDAP group sync docs may suffice as "background reading" for those less familiar with the topics in this blog

Reply:
@jasonblais Can you please take a look at the intro I just committed and let me know if it works? Edits welcome of course.